Source/MaestroThermal.cpp

```diff
@@ -46,7 +46,20 @@ void Maestro::MakeExplicitThermal(
     // turn off multigrid coarsening since no actual solve is performed
     info.setMaxCoarseningLevel(0);
 
-    MLABecLaplacian mlabec(geom, grids, dmap, info);
+    amrex::Vector<amrex::Geometry> small_geom(finest_level+1);
+    amrex::Vector<amrex::BoxArray> small_grids(finest_level+1);
+    amrex::Vector<amrex::DistributionMapping> small_dmap(finest_level+1);
+    amrex::Vector<amrex::MultiFab> small_rhcc(finest_level+1);
```

Comment:
small_rhcc doesn't appear to be used.  delete all 4 instances in MaestroThermal.cpp